Lib/test/test_doctest/test_doctest.py

```diff
@@ -133,11 +136,14 @@ def __init__(self, val=0):
             0
             """
             self.val = val
+
```

Comment:
Nit: do we need to add all the blank lines in this file? It would be nice to minimise the diff a little, I think